.github/workflows/build_gdextension.yml

```diff
@@ -0,0 +1,105 @@
+name: 🔌 Build GDExtension libs
+
+on:
+  workflow_call:
+    inputs:
+      ref:
+        type: string
+        default: ''
+        description: 'The branch, tag or SHA to checkout (leave empty for event SHA)'
+
+  workflow_dispatch:
+    inputs:
+      ref:
+        type: string
+        default: ''
+        description: 'The branch, tag or SHA to checkout (leave empty for event SHA)'
+
+env:
+  # Default SCons flags applied to each build.
+  SCONSFLAGS: verbose=yes debug_symbols=yes symbols_visibility=visible
+
+jobs:
+  build:
+    name: ${{matrix.name}}
+    runs-on: ${{matrix.runner}}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: 🐧 Linux (x86_64, release)
+            runner: ubuntu-22.04
```

Comment:
curious why pinning 22.04 as opposed to `latest` ?

Reply:
As a rule of thumb, we want to build Linux binaries on the oldest available distro, because otherwise it may fail to run on some installations due to libc incompatibilities. I had issues with SteamDeck in the past, for example.
So maybe we should pin it at 20.04, instead :thinking: